glide.yaml

```diff
@@ -34,3 +34,11 @@ import:
   version: =1.2.0
 - package: github.com/stretchr/testify
   version: =1.2.1
+- package: github.com/docker/distribution
+  version: 16128bbac47f75050e82f7e91b04df33775e0c23
```

Comment:
Did you check to see if there are actual release versions corresponding to these SHAs? If not, they may be SHAs that represent the HEAD of a repo -- effectively between supported releases.